charts/dex.yaml

```diff
@@ -31,14 +31,14 @@ ingress:
   hosts:
     - null
 
-# extraVolumes:
-#   - name: theme
-#     configMap:
-#        name: dex-branding
+extraVolumes:
+  - name: dex-login
+    configMap:
+      name: dex-login
```

Comment:
I wish we'd have given this a less generic name like `dex-scality-theme` or whatever.

Reply:
> I wish we'd have given this a less generic name like `dex-scality-theme` or whatever.

I will rename it in the 2nd iteration of authentication.